sklearn/cluster/tests/test_k_medoids.py

```diff
@@ -0,0 +1,173 @@
+"""Testing for K-Medoids"""
+import numpy as np
+
+from sklearn.utils.testing import assert_equal
+from sklearn.utils.testing import assert_true
+from sklearn.utils.testing import assert_array_equal
+from sklearn.utils.testing import raises
+
+from sklearn.metrics.pairwise import PAIRWISE_DISTANCE_FUNCTIONS
+from sklearn.metrics.pairwise import euclidean_distances
+from sklearn.datasets import load_iris
+
+# With Python 2.x ValueError needs to be imported
+try:
+    from exceptions import ValueError
+except ImportError:
+    pass
+
+from sklearn.cluster import KMedoids, KMeans
+
```

Comment:
You code does not comply with [PEP8](https://www.python.org/dev/peps/pep-0008/#blank-lines) convention.
You can easily spot required modifications with a modul like pep8 or flake8.
```
$ pip install pep8
$ pep8 path/to/module.py
```